Sources/Sentry/SentryFormatter.m

```diff
@@ -0,0 +1,26 @@
+#import "SentryFormatter.h"
+
+@implementation SentryFormatter
```

Comment:
`m`: If we add the custom implementation, I think we should move the tests we already have for the span description to an new class `SentryFormatterTests`.